prime-angular-frontend/src/app/shared/components/contact-information-form/contact-information-form.component.ts

```diff
@@ -10,11 +10,11 @@ import { FormUtilsService } from '@core/services/form-utils.service';
 })
 export class ContactInformationFormComponent implements OnInit {
   @Input() public form: FormGroup;
-  @Input() public showSmsPhone: boolean;
+  @Input() public forPhsa: boolean;
```

Comment:
Works fine but I just wonder for a shared component, should we have a parameter named `forPhsa` that is very PHSA-specific?  Would it be overkill to provide a parameter that allows the alternative text, that is, "Please provide the email address you have provided to your Health Authority to be associated with your eForms enrolment."?  Thoughts from others?

Reply:
Yah @neophyte57  I definitely agree.  We'd want to keep shared components generic, and if there is specific text that is required to be inserted based on the current module use projection.